README.md

```diff
@@ -25,60 +25,119 @@ You can get CIRCL by fetching:
- - [CPABE](./abe/cpabe): Ciphertext-policy Attribute-based Encryption.
+- [X25519](./dh/x25519) and [X448](./dh/x448) functions. ([RFC-7748])
+- [Curve4Q](./dh/curve4q) function based on FourQ curve. ([draft-ladd-cfrg-4q](https://datatracker.ietf.org/doc/draft-ladd-cfrg-4q/))
 
-#### Post-Quantum Key Encapsulation Methods
- - [CSIDH](https://csidh.isogeny.org/): Post-Quantum Commutative Group Action
- - [Kyber](https://pq-crystals.org/kyber/) KEM: modes 512, 768, 1024
- - [FrodoKEM](https://frodokem.org/) KEM: modes 640-SHAKE
- - (**insecure, deprecated**) [SIDH/SIKE](https://sike.org/): Supersingular Key Encapsulation with primes p434, p503, p751
+| Digital Signature Schemes |
+|:---:|
 
-#### Post-Quantum Public-Key Encryption
- - [Kyber](https://pq-crystals.org/kyber/) PKE: modes 512, 768, 1024
+- [Ed25519](./sign/ed25519) and [Ed448](./sign/ed448) signatures. ([RFC-8032])
 
-#### Post-Quantum Digital Signature Schemes
- - [Dilithium](https://pq-crystals.org/dilithium/): modes 2, 3, 5
+| Prime Groups |
+|:---:|
 
-#### Field Arithmetic
- - Fp25519, Fp448, Fp381
+ - [P-256, P-384, P-521](./group). ([FIPS 186-5])
+ - [Ristretto](./group) group. ([RFC-9496])
+ - [Bilinear pairings](./ecc/bls12381): with the [BLS12-381] curve, and hash to G1 and G2.
+ - [Hash to curve](./group), hash to field, XMD and XOF [expanders](./expander). ([RFC-9380])
 
-#### Elliptic Curves
- - P-384 Curve
- - [FourQ](https://eprint.iacr.org/2015/565)
- - [Goldilocks](https://eprint.iacr.org/2015/625)
- - [BLS12-381](https://electriccoin.co/blog/new-snark-curve/): Bilinear pairings, hash to G1 and G2.
+| High-Level Protocols |
+|:---:|
+
+ - [HPKE](./hpke): Hybrid Public-Key Encryption ([RFC-9180])
+ - [VOPRF](./oprf): Verifiable Oblivious Pseudorandom functions. ([RFC-9497])
+ - [RSA Blind Signatures](./blindsign/blindrsa). ([RFC-9474])
+ - [Partilly-blind](./blindsign/blindrsa/partiallyblindrsa/) Signatures. ([draft-cfrg-partially-blind-rsa](https://datatracker.ietf.org/doc/draft-amjad-cfrg-partially-blind-rsa/))
+ - [CPABE](./abe/cpabe): Ciphertext-Policy Attribute-Based Encryption. ([ia.cr/2019/966])
+ - [OT](./ot/simot): Simplest Oblivious Transfer ([ia.cr/2015/267]).
+ - [Threshold RSA](./tss/rsa) Signatures ([Shoup Eurocrypt 2000](https://www.iacr.org/archive/eurocrypt2000/1807/18070209-new.pdf)).
+
+### Post-Quantum Cryptography
+
+| KEM: Key Encapsulation Methods |
+|:---:|
+
+ - [CSIDH](./dh/csidh): Post-Quantum Commutative Group Action ([CSIDH](https://csidh.isogeny.org/)).
+ - [Kyber KEM](./kem/kyber): modes 512, 768, 1024 ([KYBER](https://pq-crystals.org/kyber/)).
+ - [FrodoKEM](./kem/frodo): modes 640-SHAKE. ([FrodoKEM](https://frodokem.org/))
+ - (**insecure, deprecated**) ~~[SIDH/SIKE](./kem/sike)~~: Supersingular Key Encapsulation with primes p434, p503, p751 ([SIKE](https://sike.org/)).
+
+| PKE: Public-Key Encryption |
+|:---:|
+
+ - [Kyber PKE](./pke/kyber): modes 512, 768, 1024 ([Kyber](https://pq-crystals.org/kyber/)).
```

Comment:
We shouldn't list this, or add a warning that this is not secure for general purpose.

Reply:
ok, it's better removing it.